### DISCUSSION TITLE
VSCode automatically executes "npx vitepress --port=4000" when opening the Vitepress project, how to close it?

### DISCUSSION THREAD

**Discussion by aiktb** [vuejs/vitepress#2402] [Q&A] [answered]
I have been using WebStorm to develop a project, when I open a VitePress project with VScode, the terminal will automatically execute "npx vitepress --port=4000", and then generate a '.vitepress' directory in the root directory.
 I don't know why, someone Can you tell me?
![Snipaste_2023-05-22_17-19-44](https://github.com/vuejs/vitepress/assets/104723527/8cf3ccc1-ed90-4d0b-9664-340016d84998)

**Comment by brc-dd** [comment, marked as answer]
You probably have some extension installed that's doing that. Removing/disabling that should work.

**Reply by aiktb** [reply]
Oh! I installed a 'vite' plugin and didn't notice that it hadn't been updated in two years.
![Snipaste_2023-05-22_17-54-54](https://github.com/vuejs/vitepress/assets/104723527/2d5f16b1-d4e6-4673-8cc9-00530a2b4392)
